employdat-app/routes/web.php

```diff
@@ -22,17 +22,17 @@
 Route::get('/order/email', [dats_controller::class, 'orderEmail'])->name('order_email');
 Route::get('/order/post', [dats_controller::class, 'orderPost'])->name('order_post');
 Route::get('/search', [dats_controller::class, 'search'])->name('search_name');
-Route::get('/insert',function(){return view('insert');})->name('insert');
+Route::get('/insert',function(){return view('insert');})->name('insert')->middleware('auth');
```

Comment:
Instead of marking particular routes, please use middleware group to hold multiple routes under the auth.